<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/photo/eye.jpg</normaloff>:/new/photo/eye.jpg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="choise">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 50px;
  height: 40px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QPushButton" name="adminPb">
         <property name="font">
          <font>
           <family>Candara Light</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ADMIN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="picture_3">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="picture.qrc">:/photo/wait.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="customerPb">
         <property name="font">
          <font>
           <family>Candara Light</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CUSTOMER</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelChois">
         <property name="font">
          <font>
           <family>Candara Light</family>
           <pointsize>48</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>who are you?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loginAdmin">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
                 background-color: white;
                 color: black;
                 border: 1px solid rgb(110, 110, 110);
              border-radius: 10px;
                 padding: 3px 7px;
             }

             QLineEdit:focus {
                 border: 2px solid rgb(90, 90, 90);
            }QPushButton
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 50px;
  height: 40px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }</string>
      </property>
      <widget class="QLabel" name="picture">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>201</width>
         <height>461</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="picture.qrc">:/photo/oddmen.jpg</pixmap>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>301</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="systempassLb">
          <property name="text">
           <string>System password:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="systmpassLE">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nameLb">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameLE"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="passwordLb">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="passworLE">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="loginPB">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="loginCustomer">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
                 background-color: white;
                 color: black;
                 border: 1px solid rgb(110, 110, 110);
              border-radius: 10px;
                 padding: 3px 7px;
             }

             QLineEdit:focus {
                 border: 2px solid rgb(90, 90, 90);
            }QPushButton
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 50px;
  height: 40px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }</string>
      </property>
      <widget class="QLabel" name="picture_2">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>201</width>
         <height>461</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="picture.qrc">:/photo/oddmen.jpg</pixmap>
       </property>
      </widget>
      <widget class="QWidget" name="widgetCust" native="true">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>301</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameCustLE"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameCustLb">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="passworCustLE">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="loginCustPB">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passwordCustLb">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
